scss/_mixins.scss

```diff
@@ -51,7 +67,7 @@
 // 
 // Mixin to add vertical spaing to the elements in a container block
 //
-@mixin container-vertial-spacing {
+@mixin container-vertical-spacing {
```

Comment:
looks like Nelson is using the misspelling, not sure if we want to rename this. I might want to leave up the misspelling or create a new ticket for fixing that tech debt https://github.com/WPMedia/fusion-news-theme-blocks/blob/b3190b381cf28dc38f70780549cd3949c6d93da4/blocks/card-list-block/features/card-list/card-list.scss#L14

Reply:
@brandonhunty -- did we address this technical debt or talk on it? would be a smallfix to blocks